src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -241,6 +246,8 @@ trait FastSync {
 
     private def handleActorTerminate(ref: ActorRef) = {
       context unwatch ref
+      val peer = assignedHandlers(ref)
+      peerRequestsTime -= peer
```

Comment:
no done means that we finished with request and peer is free for next round (after throttle period)